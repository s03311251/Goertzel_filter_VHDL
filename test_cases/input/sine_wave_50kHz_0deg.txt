2000
29E3
32CE
39E3
3E6E
3FFF
3E6E
39E3
32CE
29E3
2000
161C
0D31
061C
0191
0000
0191
061C
0D31
161C
1FFF
29E3
32CE
39E3
3E6E
3FFF
3E6E
39E3
32CE
29E3
2000
161C
0D31
061C
0191
0000
0191
061C
0D31
161C
1FFF
29E3
32CE
39E3
3E6E
3FFF
3E6E
39E3
32CE
29E3
2000
161C
0D31
061C
0191
0000
0191
061C
0D31
161C
1FFF
29E3
32CE
39E3
3E6E
3FFF
3E6E
39E3
32CE
29E3
2000
161C
0D31
061C
0191
0000
0191
061C
0D31
161C
1FFF
29E3
32CE
39E3
3E6E
3FFF
3E6E
39E3
32CE
29E3
2000
161C
0D31
061C
0191
0000
0191
061C
0D31
161C
