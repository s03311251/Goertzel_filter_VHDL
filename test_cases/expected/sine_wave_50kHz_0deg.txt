FFEBC600
FFECC3C0
